.pipelines/singletenancy/aks/e2e-job-template.yaml

```diff
@@ -1,44 +1,89 @@
 parameters:
   name: ""
   displayName: ""
-  arch: ""  
+  arch: ""
   os: ""
   clusterType: ""
   clusterName: ""
+  vmSize: ""
   k8sVersion: ""
   windowsOsSku: ""
   os_version: ""
+  scaleup: ""
 
 stages:
   - stage: ${{ parameters.name }}
     displayName: E2E - ${{ parameters.displayName }}
-    dependsOn: 
+    variables:
+      GOPATH: "$(Agent.TempDirectory)/go" # Go workspace path
+      GOBIN: "$(GOPATH)/bin" # Go binaries path
+      modulePath: "$(GOPATH)/src/github.com/Azure/azure-container-networking"
+      dropgzVersion: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.dropgzVersion'] ]
+      cniVersion: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.cniVersion'] ]
+      npmVersion: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.npmVersion'] ]
```

Comment:
do we need this since we are not running npm as part of PR pipeline test?

Reply:
No, that can be deleted. Will take another pass to make sure that there are no variables leaking out.